src/functionBased/Components/TodoContainer.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead